### PR TITLE
ENH: Replace log_path with log_dir

### PR DESCRIPTION
**Pull request by ElDeveloper**
Log files are saved according to the port where qiita is running.

### CODE REVIEW

File: qiita_core/configuration_manager.py [hunk 1]
```diff
@@ -35,7 +35,9 @@ class ConfigurationManager(object):
     base_url: str
         base URL for the website, in the form http://SOMETHING.com
     base_data_dir : str
-        Path to the base directorys where all data file are stored
+        Path to the base directory where all data files are stored.
+    log_path : str
```
Comment (josenavas) [inline qiita_core/configuration_manager.py]: Is this `log_dir`?

Comment (ElDeveloper) [inline qiita_core/configuration_manager.py]: You are correct, fixed.

**Comment by ElDeveloper** [comment]
Tests came back with this error ... I have assumed it to be stochastic and re-ran the tests (waiting).

```
======================================================================
ERROR: test_post_artifact (qiita_pet.handlers.study_handlers.tests.test_artifact.NewArtifactHandlerTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/biocore/qiita/qiita_pet/handlers/study_handlers/tests/test_artifact.py", line 128, in test_post_artifact
    wait_for_prep_information_job(self.prep.id)
  File "/home/travis/build/biocore/qiita/qiita_core/testing.py", line 42, in wait_for_prep_information_job
    wait_for_processing_job(job_id)
  File "/home/travis/build/biocore/qiita/qiita_core/testing.py", line 60, in wait_for_processing_job
    while job.status not in ('success', 'error'):
  File "/home/travis/build/biocore/qiita/qiita_db/processing_job.py", line 262, in status
    return qdb.sql_connection.TRN.execute_fetchlast()
  File "/home/travis/build/biocore/qiita/qiita_db/sql_connection.py", line 510, in wrapper
    return func(self, *args, **kwargs)
  File "/home/travis/build/biocore/qiita/qiita_db/sql_connection.py", line 782, in execute_fetchlast
    return self.execute()[-1][0][0]
  File "/home/travis/build/biocore/qiita/qiita_db/sql_connection.py", line 510, in wrapper
    return func(self, *args, **kwargs)
  File "/home/travis/build/biocore/qiita/qiita_db/sql_connection.py", line 759, in execute
    return self._execute()
  File "/home/travis/build/biocore/qiita/qiita_db/sql_connection.py", line 706, in _execute
    self._raise_execution_error(sql, sql_args, e)
  File "/home/travis/build/biocore/qiita/qiita_db/sql_connection.py", line 651, in _raise_execution_error
    % (sql, str(sql_args), str(error)))
ValueError: Error running SQL query:
Query: SELECT processing_job_status
                     FROM qiita.processing_job_status
                        JOIN qiita.processing_job
                            USING (processing_job_status_id)
                     WHERE processing_job_id = %s
Arguments: [u'547d727d-ba61-4640-b75c-d794f5e37780']
Error: error with status PGRES_TUPLES_OK and no message from the libpq
```

**Comment by josenavas** [comment]
@antgonza I've created a new branch and @ElDeveloper will do a PR against that branch. The rationale behind this change is that I would expect the PR to fail given that the config file has changed, so the encoded config file should've raised an error. I think that didn't happen because it is not using that config file with this PR - once we merge this PR in that other branch, we can double check if it fails (or not) and we can then fix the config file. 

@ElDeveloper the branch is called `log-file-to-log-dir`.

Thanks!

**Pull request closed without merging**